### ISSUE TITLE
Cannot download bike-rentals csv

### ISSUE THREAD

**Issue by AlejandroRR94** [tvquynh/Hands-On-Gradient-Boosting-with-XGBoost-and-Scikit-learn#1]
I cloned the repo. When I open the "bike-rentals.csv"  I get that the csv contains the following:

"""
version https://git-lfs.github.com/spec/v1
oid sha256:c37fd915a375aabaf026c3b5efac46dd7490a817c1a56b6e9b8ca83d2537e261
size 68796
"""

I searched a little bit and this is caused because the storage capacity provided by the current GitHub plan was exceeded.

Was it intentional to leave it like this? If so, does anyone know how can I download the files?

Thank you very much in advanced
